packages/gestalt/src/TableSortableHeaderCell.js

```diff
@@ -102,30 +130,24 @@ export default function TableSortableHeaderCell({
           onFocus={() => setFocused(true)}
           onBlur={() => setFocused(false)}
         >
-          <Box
-            display="flex"
-            alignItems="center"
-            justifyContent={align}
-            dangerouslySetInlineStyle={{
-              __style: { flexDirection: align === 'end' ? 'row-reverse' : 'row' },
-            }}
-          >
+          <Box display="flex" alignItems="center" justifyContent={align}>
+            {align === 'end' && (
+              <SortIcon
```

Comment:
Had to do this because `row-reverse` isn't accessible

Reply:
Can you add a comments in the descripton about what not accessible means.